### DISCUSSION TITLE
Is it possible to restart the simulation without statistics file ?

### DISCUSSION THREAD

**Discussion by gauravity30** [xcompact3d/Incompact3d#215] [Q&A] [answered]
Hello,
By mistake the statistics folder got deleted, and now I want to restart the simulation from a certain time-step. Is it possible to do so ?
I have the `checkpoint` file and `restart.info` file. Is there any solution ?

**Comment by mathrack** [comment, marked as answer]
Hello,
This is not an issue, but you need to adapt the parameter `initstat`. See https://github.com/xcompact3d/Incompact3d/blob/master/src/statistics.f90#L299

**Reply by gauravity30** [reply]
Thank you for the reply !
So, I think I can bypass the statistic reading by commeting this line,
https://github.com/xcompact3d/Incompact3d/blob/a0bec26fdb5a002cfe351e3a398028aa0c93a35f/src/statistics.f90#L304

**Reply by mathrack** [reply]
There is no need to change the code. You need to change the value of the variable `initstat` in the input file.

**Reply by gauravity30** [reply]
Yeah, that also works ! Thanks a ton !
